packages/media-query/src/use-breakpoint-value.ts

```diff
@@ -19,7 +19,7 @@ import { useBreakpoint } from "./use-breakpoint"
  */
 export function useBreakpointValue<T = any>(
   values: Partial<Record<string, T>> | T[],
-  defaultBreakpoint?: string,
+  defaultBreakpoint = "base", // default value ensures SSR+CSR consistency
```

Comment:
If `defaultBreakpoint` is set by default, layout flashing will probably occur even in a Vanilla environment.

Although rendering mismatch may be more serious than layout flashing...

Reply:
I think it would be better to have a global `defaultBreakPoint` setting for users who use SSR.

Reply:
Exactly! We need to support SSR. And this is by my knowledge only possible with a rerender on the client.

It's a great idea to configure this via the theme. I think it's best to do this iterative. 

1. fix the bug
2. add the theme config in another PR